build.sbt

```diff
@@ -668,8 +643,11 @@ lazy val `quill-cassandra` =
     .settings(
       Test / fork := true,
       libraryDependencies ++= Seq(
-        "com.datastax.oss"        % "java-driver-core"   % "4.14.1",
-        "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.1"
+        "com.datastax.oss" % "java-driver-core" % "4.15.0",
+        (CrossVersion.partialVersion(scalaVersion.value) match {
+          case Some((2, 12)) => "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0"
+          case _             => "org.scala-lang.modules" %% "scala-java8-compat" % "1.0.2"
+        })
```

Comment:
Akka uses `0.8.0` in its 2.12 version and `1.0.x` in its 2.13 version. 
These versions are not binary compatible